### DISCUSSION TITLE
9번째 미팅 때 논의하고 싶은 내용

### DISCUSSION THREAD

**Discussion by Joseph-Cha** [Joseph-Cha/JustFight_Client#41] [Q&A] [answered]
1. Pull Request Review
    - #35 
    - #36 
    - #37 
2. Approve 기준?
3. 마일드 스톤 관리 시작함

**Comment by jongfeel** [comment, marked as answer]
### Approve 기준?

- 더 이상 논의할 내용이 없거나, 추가로 논의해야 할 내용에 대한 피드백을 주고 받으면서 논의가 이루어졌을 때 기준

### Milestone 관리

- 내가 생각하는 일정보다 더 느슨하게, 그리고 일정이 변경되면 개의치 말고 milestone due date도 변경.
